man/import.q.sorts.Rd

```diff
@@ -17,69 +18,75 @@
+	These hash values change whenever \emph{anything} in the full item wordings is changed, and allow a precise idenfication of different versions of an item.
 
-Q-sorts are conveniently entered as \code{*.csv} (comma separated values) in popular spreadsheet editors.
-This function will ignore any rows in the files below the maximum height of columns expected from the q-distribution.
+	Q sorts are conveniently entered as *.CSV (comma separated values) files in popular spreadsheet editors.
+	This function will ignore any rows in the files below the maximum height of columns expected from the \code{q.distribution}.
 
-This function includes a number of tests to verify the integrity of entered q-sorts:
-\enumerate{
-	\item \code{manual.lookup} tables provided are tested for duplicate identifiers.
-	\item Function returns a warning if some participants do not have q-sort-files under all conditions (applies only if there are more than one conditions).
-	\item Function errors out if there are identifiers in a q-sort not defined in the (automatic or manual) lookup table.
-	\item Function errors out if the distribution in a q-sort does not conform to the defined \code{q.distribution}.
-	\item Function errors out if there are items in the sample \code{q.set} that cannot be found in a q-sort.
-	\item Function errors out if there are items in a q-sort that cannot be found in the sample \code{q.set}.
-}
+	This function includes a number of tests to verify the integrity of entered Q sorts:
+	\enumerate{
+		\item \code{manual.lookup} tables provided are tested for duplicate identifiers.
+		\item Function returns a warning if some participants do not have Q sort files under all conditions (applies only if there are more than one \code{conditions}).
+		\item Function errors out if there are IDs in a Q sort not defined in the (automatic or manual) lookup table.
+		\item Function errors out if the distribution in any given Q sort does not conform to the defined \code{q.distribution}.
+		\item Function errors out if there are items in the sample \code{q.set} that cannot be found in any given Q sort.
+		\item Function errors out if there are items in any given Q sort that cannot be found in the sample \code{q.set}.
+	}
+
+	More details on the files and directory structure expected by this function can be found in the \link{qmethod-package} documentation, and on the package \href{https://github.com/aiorazabala/qmethod/wiki/Data-management}{wiki}.
 }
 
 \value{
-	Returns a matrix or array (of more than one condition) with short handles as rownames, people as colnames, conditions (if more than one) as 3rd dimname and ranks in cells, as expected for analysis by \code{\link{qmethod}}.
+	Returns a matrix or array (of more than one condition) with short handles as rownames, people as colnames, conditions (if more than one) as 3rd dimension and ranks in cells, as expected for analysis by \code{\link{qmethod}}.
```

Comment:
`Returns a matrix (when there is a single condition) or array (with 2 or more conditions) with short handles as row names, people as column names, conditions (if more than one) as 3rd dimension and ranks in cells, as expected for analysis by \code{\link{qmethod}}.`

Does `qmethod()` work if you simply input such an array?